src/main/java/com/google/javascript/gents/CollectModuleMetadata.java

```diff
@@ -192,9 +195,9 @@ void registerProvidesModule(Node n, String file, String namespace) {
     private boolean hasImports = false;
 
     /** namespace of the module in the original closure javascript. */
```

Comment:
I always find the comments in this code confusing because of "namespace" and "module" words used somewhat ad-hoc. Can you take a stab at improving the comments here. For example, adding that we consider goog.module and goog.provides - "namespaces" for the purposes of this calculation.